src/components/v2/V2Create/forms/FundingForm/index.tsx

```diff
@@ -144,16 +148,38 @@ export default function FundingForm({
   } = useMemo(() => {
     const now = new Date().valueOf() / 1000
 
+    // Only locked if the split exists in the saved configuration (from projectContext)
+    // We want to be able to edit splits with lockedSplits that were just added
+    // or reedit splits whose lockedSplit were edited in a reconfig
+    const confirmedSplitsIncludesSplit = (split: Split) => {
```

Comment:
Checks if the given split exists (exactly - isEqual) in the projectContext. If it doesn't, then it means it was just added or edited is which case we want to still be able to edit it